metrics_handler/README.md

```diff
@@ -85,7 +85,12 @@ Full config options:
   },
 
   # (Optional) Defaults to True. Set to false to disable all BigQuery writes.
-  "write_to_bigquery": "True"
+  "write_to_bigquery": "True",
```

Comment:
Up above in this file it lists 'Available strategies:'
Could you add "average" to that list?

Reply:
Done.